commands/curl_post.go

```diff
@@ -68,6 +68,8 @@ func init() {
 	curlPostCmd.Flags().StringP(
 		curlDataFlagName, "d", "",
 		"Data for the REST API. If value starts with '@', the rest should be a file name to read the data from.")
+	curlPostCmd.Flags().StringP(curlFormDataFileFlagName, "F", "", "Specify multipart upload file. This causes curl to POST data using the Content- Type multipart/form-data according to RFC2388."+
```

Comment:
Is the file type required to be json? If so, lets specify it in this command description?

Reply:
not necessarily. depending on what the opensearch / kibana rest api expects.
the form-data-file flag parses the file in bytes. what is in the file, or what format it has, does not matter. it is only checked whether the file exists. 😃